sentry-android-fragment/src/main/java/io/sentry/android/fragment/SentryFragmentLifecycleCallbacks.kt

```diff
@@ -57,7 +57,11 @@ class SentryFragmentLifecycleCallbacks(
     ) {
         addBreadcrumb(fragment, "created")
 
-        startTracing(fragment)
+        // we only start the tracing for the fragment if the fragment has been added to its activity
+        // and not only to the backstack
+        if (fragment.isAdded) {
```

Comment:
just curious if we should just call `startTracing` inside `onAttach`?

Reply:
`onFragmentAttached` is called even before `onFragmentCreated`

Reply:
and also called by non added fragments